rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -63,6 +63,26 @@ def create_bucket(bucket_name, rgw, user_info, location=None):
             raise TestExecError("bucket creation failed")
     else:
         raise TestExecError("bucket creation failed")
+
+    is_multisite = utils.is_cluster_multisite()
```

Comment:
@ckulal , can we have this as a method say **get_remote_ip**  in reusable.py, that we call the method here and as well in the user_create.py file.

Reply:
Done @mkasturi18 
logs: http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/sync_status_enhance/retry/